src/sentry/models/groupenvironment.py

```diff
@@ -9,24 +9,29 @@ class GroupEnvironment(Model):
 
     group_id = BoundedPositiveIntegerField()
     environment_id = BoundedPositiveIntegerField()
+    first_release_id = BoundedPositiveIntegerField(null=True)
 
     class Meta:
         app_label = 'sentry'
         db_table = 'sentry_groupenvironment'
+        index_together = [
+            ('environment_id', 'first_release_id'),
```

Comment:
This seems like a sensible ordering since it facilitates searching groups by environment as well as groups by first release in an environment. Non-environment specific first release tracking [already exists on the `Group` model](https://github.com/getsentry/sentry/blob/3a9695bd223082de497b725aa62b4571f08b5e8d/src/sentry/models/group.py#L205) and would be difficult to use here since it's already broken down by environment.